README.md

```diff
@@ -23,13 +27,98 @@ bundle add sincerely
 rails g sincerely:install
 ```
 
-2. Now let's generate and run a migration to create the `notifications` and `notification_templates` tables and the `Notification` and the `NotificationTemplate` model:
+2. You need to generate and run a migration to create the `notifications` and `notification_templates` tables and the `Notification` and the `NotificationTemplate` model:
 ```bash
 rails g sincerely:migration Notification
 
 rails db:migrate
 ```
 
+3. If you want to enable event callbacks you need to run the `events` task to create the `sincerely_delivery_events` table and the `Sincerely::DeliveryEvent` model:
+```bash
+rails g sincerely:events
+
+rails db:migrate
+```
+
+## Configuration
+1. You need to set the notification model generated in the previous step by setting the `notification_model_name` option in the `config/sincerely.yml` file:
+
+```
+delivery_methods:
+  email:
+    class_name: Sincerely::DeliveryMethods::EmailAwsSes
+    options:
+  sms:
+```
+
+2. You need to set the delivery method for each delivery options (email/sms/push) by setting the `delivery_methods` option in the `config/sincerely.yml` file:
+
+```
+notification_model_name: Notification
+```
+
+## Usage
+1. For a detailed description of configuring AWS SES email sending see the following blog post:
```

Comment:
I assume the blog post link is being added later, right?

Reply:
Or is this blogpost an AWS reference?

Reply:
> I assume the blog post link is being added later, right?

correct